src/core/ddsc/src/dds_sertype_builtintopic.c

```diff
@@ -24,7 +24,7 @@
 #include "dds/ddsi/ddsi_typelookup.h"
 #include "dds__serdata_builtintopic.h"
 
-/* FIXME: sertopic /= ddstopic so a lot of stuff needs to be moved here from dds_topic.c and the free function needs to be implemented properly */
+/* FIXME: sertype /= ddstopic so a lot of stuff needs to be moved here from dds_topic.c and the free function needs to be implemented properly */
```

Comment:
This looks like a weird leftover from something. A FIXME that doesn't make sense is best removed.

Reply:
Removed in [19ca697](https://github.com/eclipse-cyclonedds/cyclonedds/pull/1397/commits/19ca697cdd23995b85f569d55ce3e6ac6b249cc4)